add enum docs (#379)
Laravel-Backpack/docs: 5.x/crud-columns.md

<hr>

<a name="enum"></a>
### enum

> The enum column will output the result from your enums (database or php enums).

Reviewer: ```suggestion
The enum column will output the value of your database ENUM column or your PHP enum attribute.
```
